.github/workflows/test_release_conda.yml

```
@@ -0,0 +1,68 @@
+name: Test Conda-forge release
+on:
+  workflow_call:
+    inputs:
+      version:
+        description: 'The Euphonic release version to test e.g. 0.6.1'
+        required: true
+        type: string
+
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'The Euphonic release version to test e.g. 0.6.1'
+        required: true
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest, macos-13]
+        python-version: ['3.10', '3.11', '3.12']
+      fail-fast: false
```

I know it's *exactly* the same thing, but I find it a bit more legible.
```suggestion
      fail-fast: false
      matrix:
        os: [ubuntu-latest, windows-latest, macos-latest, macos-13]
        python-version: ['3.10', '3.11', '3.12']
```

---

I see what you mean!